flow-typed/box-ui-elements.js

```diff
@@ -352,20 +352,22 @@ type JSONPatch = {
 type JSONPatchOperations = Array<JSONPatch>;
 
 type BoxItemVersion = {
-    action: 'upload' | 'delete' | 'restore',
     authenticated_download_url?: string,
     collaborators?: Object,
     created_at: string,
     extension?: string,
     id: string,
     is_download_available?: boolean,
     modified_at?: string,
-    modified_by: User,
+    modified_by: ?User,
     name?: string,
     permissions?: BoxItemVersionPermission,
+    restored_at?: string,
+    restored_by?: ?User,
```

Comment:
Doesn't this have a default value of the placeholder user? If so, this shouldn't be a maybe type? 🍹

Reply:
It doesn't, unfortunately. Its value can be explicitly `null` in the endpoint response.